### ISSUE TITLE
cert_dates() for Alpine/busybox does not recognise revoke_status() input date

### ISSUE THREAD

**Issue by TinCanTech** [OpenVPN/easy-rsa#592]
`revoke_status()` input date format is: `22-12-25 00:00:01Z`
`cert_dates()` expects `"%b %e %H:%M:%S %Y"`

`cert_dates()` requires:
```
	# Alpine Linux and busybox - alternate
	elif expire_date_s="$(date -D "%y-%m-%d %H:%M:%S" -d "$crt_not_after" +%s 2>/dev/null)"
	then
        ...
```

**Comment by TinCanTech** [comment]
Linking #593

**Comment by TinCanTech** [comment]
Closed via #595 - Problem code no longer exists.
